src/pkg/bundle/pull.go

```diff
@@ -25,11 +26,9 @@ import (
 // Pull pulls a bundle and saves it locally
 func (b *Bundle) Pull() error {
 	ctx := context.TODO()
-	// use uds-cache/packages as the dst dir for the pull to get auto caching
-	// we use an ORAS ocistore to make that dir look like an OCI artifact
-	cacheDir := filepath.Join(zarfConfig.GetAbsCachePath(), "packages")
-	if err := helpers.CreateDirectory(cacheDir, 0o755); err != nil {
-		return err
+	tmpDstDir, err := zarfUtils.MakeTempDir(config.CommonOptions.TempDirectory)
+	if err != nil {
+		return fmt.Errorf("pull bundle unable to create temp directory: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("unable to create temp directory: %w", err)
```

❤️ the use of `%w`, TIL

Reply:
yeah, not really utilizing it right now but could help out in the future